tasks/rh-sign-image/rh-sign-image.yaml

```
@@ -0,0 +1,87 @@
+---
+apiVersion: tekton.dev/v1
+kind: Task
+metadata:
+  name: rh-sign-image
+  labels:
+    app.kubernetes.io/version: "0.1.0"
+  annotations:
+    tekton.dev/pipelines.minVersion: "0.12.1"
+    tekton.dev/tags: release
+spec:
+  description: >-
+    Task to create internalrequests to sign snapshot components
+  params:
+    - name: snapshotPath
+      description: Path to the JSON string of the mapped Snapshot spec in the data workspace
+      type: string
+      default: "mapped_snapshot.json"
+    - name: dataPath
+      description: Path to the JSON string of the merged data to use in the data workspace
+      type: string
+      default: "data.json"
+    - name: requester
+      type: string
+      description: Name of the user that requested the signing, for auditing purposes
+    - name: requestTimeout
+      type: string
+      default: "180"
+      description: InternalRequest timeout
+  workspaces:
+    - name: data
+      description: workspace to read and save files
+  steps:
+    - name: sign-image
+      image: quay.io/hacbs-release/release-utils:a9603c0ae1982be5e1142d44765b1475adc7ad40
+      script: |
+        #!/usr/bin/env sh
+        #
+        SNAPSHOT_PATH=$(workspaces.data.path)/$(params.snapshotPath)
+        RUNDIR="$(workspaces.data.path)/$(context.taskRun.uid)"
+
+        if [ ! -d $RUNDIR ]; then
+          mkdir -p $RUNDIR
+        fi
+
+        DATA_FILE="$(workspaces.data.path)/$(params.dataPath)"
+        if [ ! -f "${DATA_FILE}" ] ; then
+            echo "No valid data file was provided."
+            exit 1
+        fi
+
+        default_pipeline_image="quay.io/redhat-isv/operator-pipelines-images:released"
+        request=$(jq -r '.sign.request // "hacbs-signing-pipeline"' ${DATA_FILE})
+        pipeline_image=$(jq -r '.sign.pipelineImage // "${default_pipeline_image}"' ${DATA_FILE})
```

```suggestion
        pipeline_image=$(jq -r --arg default_pipeline_image $default_pipeline_image '.sign.pipelineImage // $default_pipeline_image' ${DATA_FILE})
```

this will not work. See https://github.com/redhat-appstudio/release-service-catalog/pull/246/files#diff-d3b8ab085f6a1d2f1ebdd5ebc73681bc750fe7b5eb55bcbe45851d2c086b621dL96

---

Or just make sure the bash variable gets evaluated, e.g.:
`pipeline_image=$(jq -r '.sign.pipelineImage // "'${default_pipeline_image}'"' ${DATA_FILE})`

or

`pipeline_image=$(jq -r ".sign.pipelineImage // \"${default_pipeline_image}\"" ${DATA_FILE})`